src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/DatabasesOperationsExtensions.cs

```diff
@@ -0,0 +1,1149 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
https://github.com/Azure/azure-rest-api-specs/pull/643/commits/32e59682d96a7fabac31d16706728841fcda21ef

I just wanted this to be in place in the AutoRest branch so that other people who need to build on the tests have a project to start from and can avoid the time cost of starting from nothing.  I'm not planning to release the package - that will be done by someone else after we review versions, package names, and namespaces.

Is that doable, or should I just have them work off my own fork?